src/gui/README.md

```diff
@@ -2199,3 +2141,61 @@ Result:
     "47.88.33.156:6000"
 ]
 ```
+
+### Status report for default connection peers
+
+```
+URI: /network/connections/default
+
+Method: GET
+```
+
+Example:
+
+```sh
+curl http://127.0.0.1:6420/network/connections/default
+```
+
+Response:
+
+```json
+{
+  "count": 8,
+    "total_alive": 8,
+    "total_offline": 0,
+    "Connections": [
```

Comment:
Is this the correct response? These fields should not be uppercased in the response